docs/deployment/standalone.md

```diff
@@ -36,7 +37,7 @@ In `conf/inlong.conf`, configure the parameters according to the actual situatio
 ```shell
 # local IP
 local_ip=
-# message queue: pulsar or kafka
+# message queue: pulsar or kafka or tubemq
```

Comment:
Using pulsar or kafka is more easy for `standalone mode` , so tubemq is no a recomanded.